### DISCUSSION TITLE
Error while trying to send via script from application user

### DISCUSSION THREAD

**Discussion by CryorusTheDragon** [AsamK/signal-cli#1303] [Q&A]
Encountering a little problem while trying to get a simple script running for Zabbix Monitoring using

#!/bin/bash
signal-cli -u +xxx send -m "test" --group-id "ID HERE"

Everytime I run the script I get the return that the user specified isnt registered. 
Outside of the script it works perfectly fine. 

Am I missing something obvious ?
chmod 700 and chown were run.

**Comment by alexlab2017** [comment]
It seems that te user (phone number) you are trying to send the message to is not registered, it means this number does not have Signal Messenger.

To debug you can double check the number or try a known registered user.

**Comment by AsamK** [comment]
Is that the exact script you're using? Looks like a cli parameter is unexpectedly parsed as a phone number.

**Comment by CryorusTheDragon** [comment]
Figured it out. Turns out the application this was used in doesnt call the script with user we thought but an application user.  
Changed the user from nologin to a shell login and registered the account with that user. Worked fine so far. 
NOW
When trying to send a message then it says "Failed to read local accounts list"
When I try to do it via "sudo runuser" I get " This account is currently not available

My knowledge in Linux is really limited as I only started working with it in my job for about a year and only very niche and basic things so I am standing at a wall with this.

Edit The registration on that user with a captcha was no issue
